test/test_unloader.py

```diff
@@ -16,7 +18,15 @@ class TestDbUnloader(unittest.TestCase):
     def setUp(self):
         # Create temporary directory for dirq to write to
         self.dir_path = tempfile.mkdtemp()
-        self.unloader = DbUnloader(None, self.dir_path)
+
+        # Mock out the database as we're not testing that here
+        # Note that as it's a class it needs instantiating
+        self.mock_db = mock.patch(__name__ + '.ApelMysqlDb', autospec=True, spec_set=True)
```

Comment:
What's the purpose of `__name___` here?

Reply:
Without it it fails `TypeError: Need a valid target to patch. You supplied: 'ApelMysqlDb'`. I think this is consistent with https://docs.python.org/3/library/unittest.mock.html#where-to-patch (ok, that's python3), but I'm not so very confident.